src/main/kotlin/com/getkeepsafe/dexcount/GradleApi.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2015-2016 KeepSafe Software
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.getkeepsafe.dexcount
+
+import org.codehaus.groovy.runtime.InvokerHelper
+import org.gradle.StartParameter
+
+/**
+ * Attempts to isolate our use bits of the Gradle API that have changed in
+ * incompatible ways over time.
+ */
+class GradleApi private constructor() {
```

Comment:
This will be added as an extension when I move `DexMethodCountPlugin` to `kotlin`.